packages/authentication/src/providers/auth-action.provider.ts

```diff
@@ -80,3 +86,23 @@ export class AuthenticateActionProvider implements Provider<AuthenticateFn> {
     }
   }
 }
+
+@bind(
+  asMiddleware({
+    chain: DEFAULT_MIDDLEWARE_CHAIN,
```

Comment:
@raymondfeng what's the difference between `chain` and `group`?

Reply:
We allows multiple middleware chains to be invoked in the sequence. Within each chain, the `group` is used for ordering.